src/main/resources/migration/triggers/V18__set_last_updated_trigger_for_community_offender_manager.sql

```diff
@@ -0,0 +1,27 @@
+--------------------------------------------------------------------------------
+----- Create triggers for community_offender_manager table -----
+--------------------------------------------------------------------------------
+CREATE OR REPLACE FUNCTION community_offender_manager_updated() RETURNS TRIGGER AS $$
+BEGIN
+    NEW.last_updated_timestamp = CURRENT_TIMESTAMP;
+RETURN NEW;
+END;
+$$ LANGUAGE 'plpgsql';
+
+CREATE TRIGGER community_offender_manager_updated BEFORE UPDATE
+    ON community_offender_manager FOR EACH ROW EXECUTE PROCEDURE
+    community_offender_manager_updated();
+
```

Comment:
Ok for now, but I think it easier/clearer to update the timestamp in the service code to LocalDateTime.now() and avoid any DB-specific stuff?